boards/thingy91/board.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2021 HAW-Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_thingy91
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the Thingy:91 board
+ *
+ * @author      Thanh-Viet Nguyen <TopologicRose@protonmail.com>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+#include "board.h"
+#include "stdio_base.h"
+#if defined(MODULE_LPS22HB) || defined(MODULE_HTS221)
+#include "periph/gpio.h"
```

Comment:
You can include this header unconditionally, the `#ifdef` just clutters the code